### PR TITLE
revisions. would need to update master as well

### PR DESCRIPTION
**Pull request by TOMalley104**
need to include the `storyboardID` for the TVC (done in storyboard) 
need to update any class names that are provided but do not line up with tests
**need to cherrypick spec files over from here to master**

@jmburges

### CODE REVIEW

**Comment by jmburges** [comment]
This is closing issue #2

**Comment by jmburges** [comment]
almost! This doesn't actually test that the student is setting any data in the cells. You should pass in a Mocked object and make sure the right things are in the right places

**Comment by TOMalley104** [comment]
@jmburges peep the updates por favor

File: UnitTests/Tests/FISUserSpec.m [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        textPost1.user = testUser;
+        
+        imagePost1 = [[FISImagePost alloc] init];
+        imagePost1.imageContent = [UIImage imageNamed:@"flying"];
+        imagePost1.user = testUser;
+        
+        testUser.posts = [@[textPost1, imagePost1] mutableCopy];
+    });
+    
+    it(@"has properties for username, profile picture, cover image, and posts, with no extra stuff modifying their setters",
```
Comment (jmburges) [inline UnitTests/Tests/FISUserSpec.m]: If you have an and in your `it` usually it's testing too much. This is testing *way* too much. You usually just want one, maybe two or three `expect`s per `it` block.

**Comment by jmburges** [comment]
getting closer!

File: UnitTests/Tests/FISUserSpec.m (hunk 1 above)
Comment (TOMalley104) [inline UnitTests/Tests/FISUserSpec.m]: my bad, forgot to update that spec. @jmburges: updated

**Comment by TOMalley104** [comment]
my bad, forgot to update the user spec for too many `expect`s per `it`. @jmburges: updated

**Comment by TOMalley104** [comment]
master has been updated (spec files brought over)

**Comment by jmburges** [comment]
:clap:

**Pull request merged**